win_images/win-lib.ps1

```diff
@@ -35,7 +35,7 @@ function retryInstall {
                 $pkg = @("--version", $Matches.2, $Matches.1)
             }
 
-            choco install -y --allow-downgrade --execution-timeout=300 $pkg
+            choco upgrade $pkg -y --allow-downgrade --execution-timeout=300 --source="'https://community.chocolatey.org/api/v2'"
```

Comment:
(non-blocking) Do you think it might be helpful to other non-windows-expert maintainers, to include a comment here:

```
# ahdere to chocolatey scripting best practices.
# ref: https://docs.chocolatey.org/en-us/choco/commands/#scripting-integration-best-practices-style-guide
```

Reply:
This is a good idea 👍